RecoTauTag/RecoTau/python/PFRecoTauDiscriminationAgainstElectronMVA6Phase2_cff.py

```diff
@@ -1,18 +1,18 @@
 import FWCore.ParameterSet.Config as cms
 
 # HGCal electron stuff
-from RecoEgamma.EgammaTools.cleanedEcalDrivenGsfElectronsFromMultiCl_cfi import cleanedEcalDrivenGsfElectronsFromMultiCl
+from RecoEgamma.EgammaTools.cleanedEcalDrivenGsfElectronsHGC_cfi import cleanedEcalDrivenGsfElectronsHGC
 from RecoEgamma.EgammaTools.hgcalElectronIDValueMap_cff import hgcalElectronIDValueMap
 # HGCal electrons cleaned against duplicates and electrons in barrel (pt>10GeV)
 # TauValElectronSelector defined Validation/RecoTau/plugins/Selectors.cc;
 # is there a more intuitive place where such a selector is defined?
-cleanedEcalDrivenGsfElectronsFromMultiClNoEB = cms.EDFilter('TauValElectronSelector',
+cleanedEcalDrivenGsfElectronsNoEB = cms.EDFilter('TauValElectronSelector',
```

Comment:
Should it not be named as "cleanedEcalDrivenGsfElectronsHGCNoEB" for consistency?

Reply:
@srimanob you're right, I had missed that. Fixed in the latest commit.